sources/web/datalab/polymer/modules/template-manager/template-manager.ts

```diff
@@ -92,15 +92,43 @@ class TemplateManager {
 
   public static async newNotebookFromTemplate(template: NotebookTemplate) {
 
+    const appSettings = await SettingsManager.getAppSettingsAsync();
+
+    // TODO(jimmc): Look for a user preference for tempRoot
+    let tempRoot;
```

Comment:
Why is this "temp"?

Reply:
It represents the root of where the temp file will go. I changed it to baseDir as being perhaps more clear.